pubsub/kafka/kafka.go

```diff
@@ -54,7 +54,9 @@ func (p *PubSub) Subscribe(ctx context.Context, req pubsub.SubscribeRequest, han
 		Handler:         adaptHandler(handler),
 		ValueSchemaType: valueSchemaType,
 	}
-	return p.subscribeUtil(ctx, req, handlerConfig)
+
+	p.subscribeUtil(ctx, req, handlerConfig)
```

Comment:
Do you think we should batch this?

At startup, Dapr will likely invoke `Subscribe` multiple times for different topics. This will cause the connection to Kafka to be destroyed and re-created multiple times. Perhaps here we could add a 200ms delay to batch all subscribe calls

Reply:
The code in 1.13 right now doesn't destroy the connection to Kafka, it only recreates the context that the Consume function uses. This PR should adhere to the same behavior

Reply:
Yep, this code will only close the consumer group when the pubsub component is `Close()`ed.

I think adding a batcher is a good idea. I've added one for 200ms. It uses a new struct in `kit`, please see the PR here https://github.com/dapr/kit/pull/85